pkg/daemon/daemon.go

```diff
@@ -94,14 +105,36 @@ func New(
 		rootMount:         rootMount,
 		fileSystemClient:  fileSystemClient,
 		bootedOSImageURL:  osImageURL,
-	}, nil
+	}
+
+	nodeInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
```

Comment:
No update needed, but I'm curious: Was there a specific reason to do the informer setup after the `Daemon` creation? Or is it more so it reads better to you this way rather than:

```golang
return  &Daemon{
        ....
        nodeLister:       nodeInformer.Lister(),
        nodeListerSynced: nodeInformer.Informer().HasSynced,
}, nil

```

Reply:
I noticed that elsewhere we split off the Lister, etc. (e.g. https://github.com/openshift/machine-config-operator/blob/master/pkg/controller/node/node_controller.go#L105). I can change it to your suggested method if you'd like.

Reply:
No this is fine. I was just curious :smile: